Example Apps/Example ObjC/RemoteControlManager.m

```diff
@@ -0,0 +1,248 @@
+//
+//  RemoteControlManager.m
+//  SmartDeviceLink-Example-ObjC
+//
+//  Created by Beharry, Justin (J.S.) on 8/1/22.
+//  Copyright © 2022 smartdevicelink. All rights reserved.
+//
+
+#import "RemoteControlManager.h"
+
+#import "AlertManager.h"
+#import "SmartDeviceLink.h"
+
+@interface RemoteControlManager()
+
+@property (strong, nonatomic) SDLManager *sdlManager;
+@property (assign, nonatomic, getter=isPermissionEnabled, readwrite) BOOL permissionEnabled;
+@property (strong, nonatomic) NSArray<SDLSoftButtonObject *> *homeButtons;
+
+@property (strong, nonatomic) SDLRemoteControlCapabilities *remoteControlCapabilities;
+@property (strong, nonatomic) NSString *climateModuleId;
+@property (strong, nonatomic) NSNumber<SDLBool> *hasConsent;
+@property (strong, nonatomic) SDLClimateControlData *climateData;
+@property (copy, nonatomic, readwrite) NSString *climateDataString;
+@property (strong, nonatomic, readonly) NSArray<SDLSoftButtonObject *> *remoteButtons;
+
+@end
+
+@implementation RemoteControlManager
+
+- (instancetype)initWithManager:(SDLManager *)manager hasPermission:(BOOL)permission softButtons:(NSArray<SDLSoftButtonObject *> *)buttons  {
+    self = [super init];
+    if (!self) {
+        return nil;
+    }
+    _sdlManager = manager;
+    _permissionEnabled = permission;
+    _homeButtons = buttons;
+    return self;
+}
+
+- (void)start {
+    if (![self isPermissionEnabled]) {
+        SDLLogD(@"Missing permissions for Remote Control Manager. Example remote control works only on TCP.");
```

Comment:
I just also realized that this file needs to be added to `ProxyManager logConfiguration` method to ensure that it gets a proper module in the log.